src/store/SettingsDialogStore.js

```diff
@@ -69,6 +71,7 @@ export default class SettingsDialogStore {
             showTabs: this.showTabs,
             stared: this.stared,
             setOpen: this.setOpen,
+            isFavoritable: this.isFavoritable,
```

Comment:
@brucebinary `isFavoritable` is a computed, use a mobx computed if you want to be format.
Or just use `isFavoritable: !!this.onStared` in the connect() method.

Reply:
I like the 2nd suggestion. Just implemented that. Thanks for your feedback